client/package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^7.2.1",
     "bootstrap": "^4.4.1",
     "cors": "^2.8.5",
+    "express": "^4.17.1",
```

Comment:
Is this necessary? The parent package.json already has express. And I see you bumped the express version in the parent package.json file but here in this client package.json it is the older version. This is the risk of having this in both places.

Reply:
I agree here. @DanielLaszczych, is there a reason to add `express` here? I think we should be fine removing express from the client's `package.json` and only using it in the server's `package.json`